controllers/openstackmachine_controller_test.go

```diff
@@ -413,30 +400,6 @@ func Test_reconcileDelete(t *testing.T) {
 			},
 			wantRemoveFinalizer: true,
 		},
-		{
```

Comment:
I had to remove this test, it was not passing anymore. No calls were made at all. I guess it's an upgrade issue?